superset-frontend/.storybook/main.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     builder: 'webpack5',
   },
   stories: [
+    '../packages/superset-ui-demo/storybook/**/Stories.@(tsx|jsx|mdx)',
```

Comment:
bycatch: should include stories for plugins when storybook server is run.

Reply:
Actually they are separate storybooks currently. Before merging them, we need to think about how to organize the components.